plugins/flytekit-spark/setup.py

```diff
@@ -34,4 +34,5 @@
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     scripts=["scripts/flytekit_install_spark3.sh"],
+    entry_points={"flytekit.plugins": [f"{PLUGIN_NAME}=flytekitplugins.{PLUGIN_NAME}"]},
```

Comment:
I added this because flytekit will auto load `spark` and `huggingface` structured dataset transformer. otherwise, we have to manually import transformer.

we've added the `entry_points` in vaex and polars plugin.